holidays/countries/mongolia.py

```diff
@@ -0,0 +1,183 @@
+        name = tr("Үндэсний их баяр наадам, Ардын хувьсгалын ойн баяр")
+        if self._year >= 2023:
+            self._add_holiday_jul_10(name)
+        self._add_holiday_jul_11(name)
+        self._add_holiday_jul_12(name)
+        self._add_holiday_jul_13(name)
+
+        if self._year >= 2014:
+            self._add_holiday_jul_14(name)
+            self._add_holiday_jul_15(name)
+
+        # Established on November 8th, 2012.
+        if self._year >= 2012:
+            # Genghis Khan's Birthday.
+            self._add_genghis_khan_day(tr("Их Эзэн Чингис хааны өдөр"))
+
+        # Repealed on November 8th, 2012.
+        # Re-established again on November 18th, 2016.
+        if self._year <= 2011 or self._year >= 2016:
+            # Republic Day.
+            self._add_holiday_nov_26(
+                # Republic Day.
+                tr("Бүгд Найрамдах Улс тунхагласан өдөр")
+                if self._year >= 2016
+                # Republic Holiday.
+                else tr("Бүгд Найрамдах Улс тунхагласны баяр")
+            )
+
+        # Established on August 16th, 2007.
+        # Renamed and considered a Public Holiday on December 23rd, 2011.
+        if self._year >= 2011:
+            self._add_holiday_dec_29(
+                # National Freedom and Independence Day.
+                tr("Үндэсний эрх чөлөө, тусгаар тогтнолоо сэргээсний баярын өдөр")
+            )
+
+    def _populate_workday_holidays(self):
+        # Constitutional Day of Mongolia.
+        self._add_holiday_jan_13(tr("Монгол Улсын Үндсэн хуулийн өдөр"))
+
+        # Established on December 9th, 2004.
+        if self._year >= 2005:
+            # Patriots' Day.
+            self._add_holiday_mar_1(tr("Эх орончдын өдөр"))
+
+        self._add_holiday_mar_18(
+            # Military Day.
+            tr("Монгол цэргийн өдөр")
+            if self._year >= 2011
+            # Armed Forces Day.
+            else tr("Зэвсэгт хүчний өдөр")
+        )
+
+        # Health Protection Day.
+        self._add_holiday_apr_7(tr("Эрүүл мэндийг хамгаалах өдөр"))
+
+        # Intellectual Property Day.
+        self._add_holiday_apr_26(tr("Оюуны өмчийг хамгаалах өдөр"))
+
+        # Family Day.
+        self._add_holiday_may_15(tr("Гэр бүлийн өдөр"))
+
+        # Established on June 11th, 2009
+        if self._year >= 2009:
+            # National Flag Day.
+            self._add_holiday_jul_10(tr("Монгол Улсын төрийн далбааны өдөр"))
+
+        # Youth Day.
+        self._add_holiday_aug_25(tr("Залуучуудын өдөр"))
+
+        if self._year >= 2007:
+            # New Harvest Days.
+            name = tr("Шинэ ургацын өдрүүд")
+            dt = self._add_holiday_sep_5(name)
+            for delta in range(1, 46):
+                self._add_holiday(name, _timedelta(dt, delta))
+
+        # Memorial Day of Political Defendants.
+        self._add_holiday_sep_10(tr("Улс төрийн хэлмэгдэгсдийн дурсгалын өдөр"))
+
+        # Elders' Day.
```

Comment:
```suggestion
        # Veterans' Day.
```

Reply:
I think, we can keep Elders' Day, or use Elderly Day. I agree with both options.